19.03.5.sh

```diff
@@ -511,6 +518,32 @@ do_install() {
 			)
 			exit 0
 			;;
+		sles)
+			(
+			set -x
+			$sh_c 'zypper refresh'
+			docker_pkg=$(zypper se -n -s docker | grep -E "\sdocker\s.*${docker_version//\./\\.}.*$architecture" | sort --sort=v -k3 | tail -1 | cut -d '|' -f4 | cut -d ' ' -f2)
+			if [ -z "$docker_pkg" ]; then
+			    trunc_docker_version=${docker_version%.*}
```

Comment:
Does it make sense to use the included `semverParse()` for this or not really?

Reply:
I utilized `semverParse` in the latest commit, but either approach should be okay! semverParse pulls out $major, $minor, and $patch, which we then concatenate to $major.$minor, whereas the previous substitution just removes $patch from docker_version.